src/backend/distributed/planner/query_pushdown_planning.c

```diff
@@ -1805,17 +1804,20 @@ ContainsRecurringRangeTable(List *rangeTable, RecurringTuplesType *recurType)
 							  QTW_EXAMINE_RTES_BEFORE);
 }
 
+
 bool
```

Comment:
maybe comment, e.g.

IsJsonTableRTE checks whether the RTE refers to a JSON_TABLE table function,
which was introduced in PostgreSQL 15.